Return master to releasable

alishaevn opened the pull request:
# Ticket
https://app.zenhub.com/workspaces/yul-dc-5e50083561915b11fc9e5850/issues/yalelibrary/yul-dc/565

# Expected behavior
- the gallery view of search results remains styled as the design intends
- the list view of search results looks like it did out of box, besides the title color being updated
- the yale restricted caption is in line with the title on an individual show page

# Demo
![565](https://user-images.githubusercontent.com/29032869/93115486-c5970400-f670-11ea-906a-109878e9c6b1.gif)

orangewolf commented:
@alishaevn there are a lot of changes on this PR and that makes it hard to see the ones that actually apply. Maybe you can call out the actual changes with a PR review comment in the future?

alishaevn commented:
> @alishaevn there are a lot of changes on this PR and that makes it hard to see the ones that actually apply. Maybe you can call out the actual changes with a PR review comment in the future?

@orangewolf 
makes sense.
the changes in app/assets/stylesheets/customOverrides/index_gallery.scss are the ones that make the list view work again. I just added more specificity to the classes for the gallery view styling. for example, `.document` is a class in both the list view and gallery view. but in the gallery view there's a secondary class on the same element, so I updated it to be `.document.col`. similarly, the `.caption` and `.index` classes are only on the gallery view so I added the nesting to make sure the style only affects the gallery view.

the other 2 pages are changes for the single item view. the yale restricted caption is now in line with the title on an individual show page. I kept them in this PR since the theme was to make sure that master is deployable.
